extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -253,8 +251,6 @@ public function getTrackingDimensions() {
 		$dimensions[15] = WikiaPageType::isCorporatePage() ? 'yes' : 'no';
 		$dimensions[17] = WikiFactoryHub::getInstance()->getWikiVertical( $wgCityId )['short'];
 		$dimensions[18] = $wikiCategoryNames;
-		$dimensions[23] = in_array( 'poweruser_lifetime', $powerUserTypes ) ? 'yes' : 'no';
-		$dimensions[24] = in_array( 'poweruser_frequent', $powerUserTypes ) ? 'yes' : 'no';
```

Comment:
@hakubo - please code review this part from Mercury perspective

Reply:
lgtm
should we also remove this?
https://github.com/Wikia/mobile-wiki/blob/cb593de357863cf38722e09297da97e4e10579b5/app/models/user.js#L191

Reply:
Thanks for the link - https://github.com/Wikia/mobile-wiki/pull/413